.gitignore

```diff
@@ -155,3 +155,6 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
+# Keys file
+keys.py
```

Comment:
You shouldn't need to do this, the environment variables are stored in the `Dockerfile`. You can use `os.environ.get` to get the values.